docs/COMMANDS.md

```
@@ -0,0 +1,1317 @@
+
+Complete reference for all Tuti CLI commands.
+
+---
+
+## Table of Contents
+
+- [Global Setup](#global-setup)
+  - [tuti install](#tuti-install)
+  - [tuti doctor](#tuti-doctor)
+- [Infrastructure](#infrastructure)
+  - [tuti infra:start](#tutistrastart)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)